code/controllers/subsystem/ticker.dm

```diff
@@ -598,7 +598,7 @@ SUBSYSTEM_DEF(ticker)
 		if(GANG_DESTROYED)
 			news_message = "The crew of [decoded_station_name] would like to thank the Spinward Stellar Coalition Police Department for quickly resolving a minor terror threat to the station."
 		if(SUPERMATTER_CASCADE)
-			news_message = "Universal reboot initiated on [decoded_station_name] following a major supermatter cascade."
+			news_message = "Recovery of the surviving crew of [decoded_station_name] is underway following a major supermatter cascade."
```

Comment:
This is the thing, there isn't any surviving crew, or anyone that can recover them for that matter

Reply:
i at least agree with one thing: there is no need for recovery because everyone would 100% be dead